apps/web/pages/[user].tsx

```diff
@@ -33,6 +33,7 @@ import type { inferSSRProps } from "@lib/types/inferSSRProps";
 import type { EmbedProps } from "@lib/withEmbedSsr";
 
 import PageWrapper from "@components/PageWrapper";
+import LegalNotice from "@components/ui/LegalNotice";
```

Comment:
@sean-brydon this should be in features not UI components no?

Reply:
@PeerRich i have it there because PoweredByCal is also located there.

Reply:
ok, talking to sean now where to put it. we can merge afterwards